flat/BaseRootTunnel.sol

```diff
@@ -1670,9 +1663,7 @@ abstract contract BaseRootTunnel is AccessControlMixin {
         stateSender.syncState(childTunnel, message);
     }
 
-    function _validateAndExtractMessage(bytes memory inputData) internal returns (bytes memory) {
-        require(inputData.length == 10, "RootTunnel: BAD_PAYLOAD");
```

Comment:
Why is this gone?

Reply:
This is a change rebased from `master` 88dbf0a88fd68fa11f7a3b9d36629930f6b93a05